tests/NSubstitute.Acceptance.Specs/NSubstitute.Acceptance.Specs.csproj

```diff
@@ -10,13 +18,6 @@
     <PackageReference Include="NUnit3TestAdapter" Version="3.8.0-alpha1" />
```

Comment:
Is there any reason why the alpha version is referenced here? [Version 3.8.0](https://www.nuget.org/packages/NUnit3TestAdapter/) is already released, so maybe we should upgrade :)?

Reply:
Nice catch, fixed now. Thanks!